src/k8s/pkg/k8sd/database/cluster_config.go

```diff
@@ -10,9 +10,22 @@ import (
 	"github.com/canonical/microcluster/v2/cluster"
 )
 
-var clusterConfigsStmts = map[string]int{
-	"insert-v1alpha2": MustPrepareStatement("cluster-configs", "insert-v1alpha2.sql"),
-	"select-v1alpha2": MustPrepareStatement("cluster-configs", "select-v1alpha2.sql"),
+const (
+	clusterConfigsDir string = "cluster-configs"
+)
+
+type clusterConfigsStmtsSchema struct {
+	insertV1alpha2          int
+	insertBootstrapV1alpha2 int
+	selectV1alpha2          int
+	selectBootstrapV1alpha2 int
+}
+
+var clusterConfigsStmts = clusterConfigsStmtsSchema{
+	insertV1alpha2:          MustPrepareStatement(clusterConfigsDir, "insert-v1alpha2.sql"),
+	insertBootstrapV1alpha2: MustPrepareStatement(clusterConfigsDir, "insert-bootstrap-v1alpha2.sql"),
+	selectV1alpha2:          MustPrepareStatement(clusterConfigsDir, "select-v1alpha2.sql"),
+	selectBootstrapV1alpha2: MustPrepareStatement(clusterConfigsDir, "select-bootstrap-v1alpha2.sql"),
 }
```

Comment:
This may be more intuitive:

```suggestion
type clusterConfigsStmtsSchema map[string]int

var clusterConfigsStmts = clusterConfigsStmtsSchema{
    "insertV1alpha2":          MustPrepareStatement(clusterConfigsDir, "insert-v1alpha2.sql"),
    "insertBootstrapV1alpha2": MustPrepareStatement(clusterConfigsDir, "insert-bootstrap-v1alpha2.sql"),
    "selectV1alpha2":          MustPrepareStatement(clusterConfigsDir, "select-v1alpha2.sql"),
    "selectBootstrapV1alpha2": MustPrepareStatement(clusterConfigsDir, "select-bootstrap-v1alpha2.sql"),
}
```